CRITICAL - 2024-02-27 03:51:10 --> CodeIgniter\Exceptions\FrameworkException: The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 252.
 1 SYSTEMPATH\CodeIgniter.php(252): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(214): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-02-27 03:51:11 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\erp.local\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(258): include()
#1 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\erp.local\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\erp.local\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(252): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(214): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\erp.local\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-02-27 02:54:37 --> ParseError: syntax error, unexpected variable "$routes", expecting "function" or "const"
in APPPATH\Config\Routing.php on line 114.
 1 SYSTEMPATH\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\erp.local\\app\\Config\\Routing.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Routing')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Routing')
 4 SYSTEMPATH\Config\Factories.php(280): class_exists('Config\\Routing')
 5 SYSTEMPATH\Config\Factories.php(199): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Routing')
 6 SYSTEMPATH\Config\Factories.php(158): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Routing', [])
 7 SYSTEMPATH\Common.php(215): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Config\Services.php(618): config('Config\\Routing')
 9 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::routes(false)
10 SYSTEMPATH\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('routes', [...])
11 SYSTEMPATH\Config\Services.php(615): CodeIgniter\Config\BaseService::getSharedInstance('routes')
12 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::routes()
13 APPPATH\Config\Events.php(50): CodeIgniter\Config\BaseService::__callStatic('routes', [])
14 SYSTEMPATH\Events\Events.php(154): CodeIgniter\Events\Events::Config\{closure}()
15 SYSTEMPATH\CodeIgniter.php(442): CodeIgniter\Events\Events::trigger('pre_system')
16 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
17 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-27 02:54:37 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\erp.local\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(258): include()
#1 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\erp.local\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ParseError), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\erp.local\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(ParseError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#5 {main}
  thrown
【Previous Exception】
ParseError
syntax error, unexpected variable "$routes", expecting "function" or "const"
#0 C:\xampp\htdocs\erp.local\system\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\erp.local\system\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Routing')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Routing')
#3 C:\xampp\htdocs\erp.local\system\Config\Factories.php(280): class_exists('Config\\Routing')
#4 C:\xampp\htdocs\erp.local\system\Config\Factories.php(199): CodeIgniter\Config\Factories::locateClass(Array, 'Config\\Routing')
#5 C:\xampp\htdocs\erp.local\system\Config\Factories.php(158): CodeIgniter\Config\Factories::getDefinedInstance(Array, 'Config\\Routing', Array)
#6 C:\xampp\htdocs\erp.local\system\Common.php(215): CodeIgniter\Config\Factories::__callStatic('config', Array)
#7 C:\xampp\htdocs\erp.local\system\Config\Services.php(618): config('Config\\Routing')
#8 C:\xampp\htdocs\erp.local\system\Config\BaseService.php(258): CodeIgniter\Config\Services::routes(false)
#9 C:\xampp\htdocs\erp.local\system\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('routes', Array)
#10 C:\xampp\htdocs\erp.local\system\Config\Services.php(615): CodeIgniter\Config\BaseService::getSharedInstance('routes')
#11 C:\xampp\htdocs\erp.local\system\Config\BaseService.php(258): CodeIgniter\Config\Services::routes()
#12 C:\xampp\htdocs\erp.local\app\Config\Events.php(50): CodeIgniter\Config\BaseService::__callStatic('routes', Array)
#13 C:\xampp\htdocs\erp.local\system\Events\Events.php(154): CodeIgniter\Events\Events::Config\{closure}()
#14 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(442): CodeIgniter\Events\Events::trigger('pre_system')
#15 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#17 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-02-27 02:54:55 --> ParseError: syntax error, unexpected variable "$this", expecting "function" or "const"
in APPPATH\Config\Routing.php on line 114.
 1 SYSTEMPATH\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\erp.local\\app\\Config\\Routing.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Routing')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Routing')
 4 SYSTEMPATH\Config\Factories.php(280): class_exists('Config\\Routing')
 5 SYSTEMPATH\Config\Factories.php(199): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Routing')
 6 SYSTEMPATH\Config\Factories.php(158): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Routing', [])
 7 SYSTEMPATH\Common.php(215): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Config\Services.php(618): config('Config\\Routing')
 9 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::routes(false)
10 SYSTEMPATH\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('routes', [...])
11 SYSTEMPATH\Config\Services.php(615): CodeIgniter\Config\BaseService::getSharedInstance('routes')
12 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::routes()
13 APPPATH\Config\Events.php(50): CodeIgniter\Config\BaseService::__callStatic('routes', [])
14 SYSTEMPATH\Events\Events.php(154): CodeIgniter\Events\Events::Config\{closure}()
15 SYSTEMPATH\CodeIgniter.php(442): CodeIgniter\Events\Events::trigger('pre_system')
16 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
17 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-27 02:54:55 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\erp.local\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(258): include()
#1 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\erp.local\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ParseError), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\erp.local\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(ParseError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#5 {main}
  thrown
【Previous Exception】
ParseError
syntax error, unexpected variable "$this", expecting "function" or "const"
#0 C:\xampp\htdocs\erp.local\system\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\erp.local\system\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Routing')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Routing')
#3 C:\xampp\htdocs\erp.local\system\Config\Factories.php(280): class_exists('Config\\Routing')
#4 C:\xampp\htdocs\erp.local\system\Config\Factories.php(199): CodeIgniter\Config\Factories::locateClass(Array, 'Config\\Routing')
#5 C:\xampp\htdocs\erp.local\system\Config\Factories.php(158): CodeIgniter\Config\Factories::getDefinedInstance(Array, 'Config\\Routing', Array)
#6 C:\xampp\htdocs\erp.local\system\Common.php(215): CodeIgniter\Config\Factories::__callStatic('config', Array)
#7 C:\xampp\htdocs\erp.local\system\Config\Services.php(618): config('Config\\Routing')
#8 C:\xampp\htdocs\erp.local\system\Config\BaseService.php(258): CodeIgniter\Config\Services::routes(false)
#9 C:\xampp\htdocs\erp.local\system\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('routes', Array)
#10 C:\xampp\htdocs\erp.local\system\Config\Services.php(615): CodeIgniter\Config\BaseService::getSharedInstance('routes')
#11 C:\xampp\htdocs\erp.local\system\Config\BaseService.php(258): CodeIgniter\Config\Services::routes()
#12 C:\xampp\htdocs\erp.local\app\Config\Events.php(50): CodeIgniter\Config\BaseService::__callStatic('routes', Array)
#13 C:\xampp\htdocs\erp.local\system\Events\Events.php(154): CodeIgniter\Events\Events::Config\{closure}()
#14 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(442): CodeIgniter\Events\Events::trigger('pre_system')
#15 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#17 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-02-27 02:55:03 --> ParseError: syntax error, unexpected variable "$this", expecting "function" or "const"
in APPPATH\Config\Routing.php on line 114.
 1 SYSTEMPATH\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\erp.local\\app\\Config\\Routing.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Routing')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Routing')
 4 SYSTEMPATH\Config\Factories.php(280): class_exists('Config\\Routing')
 5 SYSTEMPATH\Config\Factories.php(199): CodeIgniter\Config\Factories::locateClass([...], 'Config\\Routing')
 6 SYSTEMPATH\Config\Factories.php(158): CodeIgniter\Config\Factories::getDefinedInstance([...], 'Config\\Routing', [])
 7 SYSTEMPATH\Common.php(215): CodeIgniter\Config\Factories::__callStatic('config', [])
 8 SYSTEMPATH\Config\Services.php(618): config('Config\\Routing')
 9 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::routes(false)
10 SYSTEMPATH\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('routes', [...])
11 SYSTEMPATH\Config\Services.php(615): CodeIgniter\Config\BaseService::getSharedInstance('routes')
12 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::routes()
13 APPPATH\Config\Events.php(50): CodeIgniter\Config\BaseService::__callStatic('routes', [])
14 SYSTEMPATH\Events\Events.php(154): CodeIgniter\Events\Events::Config\{closure}()
15 SYSTEMPATH\CodeIgniter.php(442): CodeIgniter\Events\Events::trigger('pre_system')
16 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
17 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-27 02:55:03 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\erp.local\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(258): include()
#1 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\erp.local\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ParseError), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\erp.local\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(ParseError), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#5 {main}
  thrown
【Previous Exception】
ParseError
syntax error, unexpected variable "$this", expecting "function" or "const"
#0 C:\xampp\htdocs\erp.local\system\Autoloader\Autoloader.php(288): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\erp.local\system\Autoloader\Autoloader.php(266): CodeIgniter\Autoloader\Autoloader->loadInNamespace('Config\\Routing')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('Config\\Routing')
#3 C:\xampp\htdocs\erp.local\system\Config\Factories.php(280): class_exists('Config\\Routing')
#4 C:\xampp\htdocs\erp.local\system\Config\Factories.php(199): CodeIgniter\Config\Factories::locateClass(Array, 'Config\\Routing')
#5 C:\xampp\htdocs\erp.local\system\Config\Factories.php(158): CodeIgniter\Config\Factories::getDefinedInstance(Array, 'Config\\Routing', Array)
#6 C:\xampp\htdocs\erp.local\system\Common.php(215): CodeIgniter\Config\Factories::__callStatic('config', Array)
#7 C:\xampp\htdocs\erp.local\system\Config\Services.php(618): config('Config\\Routing')
#8 C:\xampp\htdocs\erp.local\system\Config\BaseService.php(258): CodeIgniter\Config\Services::routes(false)
#9 C:\xampp\htdocs\erp.local\system\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('routes', Array)
#10 C:\xampp\htdocs\erp.local\system\Config\Services.php(615): CodeIgniter\Config\BaseService::getSharedInstance('routes')
#11 C:\xampp\htdocs\erp.local\system\Config\BaseService.php(258): CodeIgniter\Config\Services::routes()
#12 C:\xampp\htdocs\erp.local\app\Config\Events.php(50): CodeIgniter\Config\BaseService::__callStatic('routes', Array)
#13 C:\xampp\htdocs\erp.local\system\Events\Events.php(154): CodeIgniter\Events\Events::Config\{closure}()
#14 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(442): CodeIgniter\Events\Events::trigger('pre_system')
#15 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#16 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#17 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-02-27 02:55:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-02-27 02:55:18 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'erp' in C:\xampp\htdocs\erp.local\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'erp', 3306, '', 0)
#1 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\erp.local\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\erp.local\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\erp.local\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\erp.local\app\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(941): App\Controllers\Login->index()
#8 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'erp' in C:\xampp\htdocs\erp.local\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\erp.local\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\erp.local\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\erp.local\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\erp.local\app\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(941): App\Controllers\Login->index()
#7 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-02-27 02:55:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'erp'
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `configuracoes`
WHERE `id_config` = :id_config:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-27 02:55:18 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\erp.local\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(258): include()
#1 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\erp.local\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\erp.local\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Database\Exceptions\DatabaseException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 8)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Database\Exceptions\DatabaseException
Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'erp'
#0 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\erp.local\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\erp.local\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\erp.local\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\erp.local\app\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(941): App\Controllers\Login->index()
#6 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-02-27 02:55:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-02-27 02:55:28 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'erp' in C:\xampp\htdocs\erp.local\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'erp', 3306, '', 0)
#1 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\erp.local\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\erp.local\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\erp.local\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\erp.local\app\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(941): App\Controllers\Login->index()
#8 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'erp' in C:\xampp\htdocs\erp.local\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\erp.local\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\erp.local\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\erp.local\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\erp.local\app\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(941): App\Controllers\Login->index()
#7 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-02-27 02:55:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'erp'
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `configuracoes`
WHERE `id_config` = :id_config:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->index()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-27 02:55:28 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\erp.local\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(258): include()
#1 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\erp.local\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\erp.local\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Database\Exceptions\DatabaseException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 8)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Database\Exceptions\DatabaseException
Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'erp'
#0 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\erp.local\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\erp.local\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\erp.local\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#4 C:\xampp\htdocs\erp.local\app\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
#5 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(941): App\Controllers\Login->index()
#6 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-02-27 02:55:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-02-27 02:55:47 --> mysqli_sql_exception: Table 'erp.configuracoes' doesn't exist in C:\xampp\htdocs\erp.local\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\erp.local\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\erp.local\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\erp.local\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\erp.local\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\erp.local\app\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(941): App\Controllers\Login->index()
#8 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-02-27 02:55:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'erp.configuracoes' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `configuracoes`
WHERE `id_config` = :id_config:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->index()
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-27 02:55:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'erp.configuracoes' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `configuracoes`
WHERE `id_config` = 1
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `configuracoes`
WHERE `id_config` = 1
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `configuracoes`
WHERE `id_config` = :id_config:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->index()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-27 02:55:47 --> [Caused by] mysqli_sql_exception: Table 'erp.configuracoes' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('SELECT *
FROM `configuracoes`
WHERE `id_config` = 1
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `configuracoes`
WHERE `id_config` = 1
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `configuracoes`
WHERE `id_config` = 1
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `configuracoes`
WHERE `id_config` = :id_config:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Login->index()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-02-27 02:55:47 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\erp.local\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(258): include()
#1 C:\xampp\htdocs\erp.local\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\erp.local\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\erp.local\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Database\Exceptions\DatabaseException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 8)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Database\Exceptions\DatabaseException
Table 'erp.configuracoes' doesn't exist
#0 C:\xampp\htdocs\erp.local\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#1 C:\xampp\htdocs\erp.local\system\Model.php(270): CodeIgniter\Database\BaseBuilder->get()
#2 C:\xampp\htdocs\erp.local\system\BaseModel.php(676): CodeIgniter\Model->doFirst()
#3 C:\xampp\htdocs\erp.local\app\Controllers\Login.php(45): CodeIgniter\BaseModel->first()
#4 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(941): App\Controllers\Login->index()
#5 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\xampp\htdocs\erp.local\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\erp.local\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-02-27 02:57:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:57:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-02-27 02:58:02 --> [DEPRECATED] Creation of dynamic property App\Controllers\Produtos::$permissao is deprecated in APPPATH\Controllers\Produtos.php on line 61.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\Produtos->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-02-27 02:58:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-02-27 02:58:04 --> [DEPRECATED] Creation of dynamic property App\Controllers\Produtos::$permissao is deprecated in APPPATH\Controllers\Produtos.php on line 61.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\Produtos->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-02-27 02:58:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-02-27 02:58:22 --> [DEPRECATED] Creation of dynamic property App\Controllers\Produtos::$permissao is deprecated in APPPATH\Controllers\Produtos.php on line 61.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\Produtos->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-02-27 02:58:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-02-27 02:58:22 --> [DEPRECATED] Creation of dynamic property App\Controllers\Produtos::$permissao is deprecated in APPPATH\Controllers\Produtos.php on line 61.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\Produtos->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-02-27 02:58:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:58:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:59:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 02:59:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-02-27 02:59:18 --> [DEPRECATED] Creation of dynamic property App\Controllers\Produtos::$permissao is deprecated in APPPATH\Controllers\Produtos.php on line 61.
 1 SYSTEMPATH\CodeIgniter.php(913): App\Controllers\Produtos->__construct()
 2 SYSTEMPATH\CodeIgniter.php(493): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-02-27 02:59:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:02:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:02:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:02:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:02:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:02:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-02-27 03:05:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
